internal/api/helpers.go

```
@@ -2,9 +2,14 @@ package api
 
 import (
 	feemodel "github.com/bsv-blockchain/go-sdk/transaction/fee_model"
+	"github.com/ccoveille/go-safecast"
 )
 
 func FeesToFeeModel(minMiningFee float64) *feemodel.SatoshisPerKilobyte {
 	satoshisPerKB := int(minMiningFee * 1e8)
-	return &feemodel.SatoshisPerKilobyte{Satoshis: uint64(satoshisPerKB)}
+	satoshisPerKBuint64, err := safecast.ToUint64(satoshisPerKB)
+	if err != nil {
+		return nil
```

In this case we should return an error